Remove try it out sections from learn (#39810)
mdn/content: files/en-us/learn_web_development/core/structuring_content/html_forms/index.md

Let's look at a basic example that includes the above three items. This form could be used to ask for a user's name and email, to sign them up for a newsletter (don't worry — it isn't connected to any server, so currently won't do anything).

```html live-sample___form-anatomy
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <title>First form</title>
  </head>
  <body>
    <form action="./submit_page" method="get">
      <h2>Subscribe to our newsletter</h2>
      <p>
        <label for="name">Name (required):</label>
        <input type="text" name="name" id="name" required />
      </p>
      <p>
        <label for="email">Email (required):</label>
        <input type="email" name="email" id="email" required />
      </p>
      <p>
        <button>Sign me up!</button>
      </p>
    </form>
  </body>
</html>
```

This is rendered as follows:

> {{EmbedLiveSample("form-anatomy", "100%", "200")}}

Reviewer: Let's make it break on purpose, but still use validation:

```suggestion
{{EmbedLiveSample("form-anatomy", "100%", "200", , , , , "allow-forms")}}
```

> Don't try to submit the form for now; you'll see it in action soon enough. To follow along with the next sections, copy the HTML code above into a new HTML file using your [code editor](/en-US/docs/Learn_web_development/Getting_started/Environment_setup/Code_editors) and open it in a new browser tab.

Reviewer: ```suggestion
If you click "Sign me up!" on the form above, you'll see an error message.
We'll explain more about why that is later, but first, copy the HTML example into a new HTML file using your [code editor](/en-US/docs/Learn_web_development/Getting_started/Environment_setup/Code_editors) and open it in a new browser tab.
```

- `action`: Contains a path to the page that we want to send the submitted form data to, to be processed. Later on, after you submit the form, you'll see `/submit_page` included in the URL. You'll also get a {{HTTPStatus("404")}} error response because the page doesn't actually exist, but that's fine for now.
- `method`: Specifies the data transmission [method](/en-US/docs/Web/HTTP/Reference/Methods) you want to use for sending the form data to the server. Don't worry about this too much for now; the `get` value causes the data to be sent as parameters attached to the end of the URL.

#### Checking out the submitted data

1. Go to the example in the separate tab, try entering a name of "Bob" and an email address of "bob@bob.com".
2. Press the `<button>`.

The `action` and `method` attributes cause the form data to be submitted in a URL along the following lines:

> `/some/url/submit_page?name=Bob&email=bob%40bob.com`

Reviewer: ````suggestion
```plain
/some/url/submit_page?name=Bob&email=bob%40bob.com
```
````

In the case of radio buttons, you generally need to provide the value that would be submitted if it is selected inside a specific `value` attribute. Note that you _can_ specify a `value` attribute on input types like `text` and `color` — the effect is that the value is pre-filled into the form field when it is first rendered.

#### `required` and `value` attributes in action

> 1. Again, go to the example you loaded in a separate tab, and try submitting the form with no value entered in either field. You'll see an error message pop up next to the "Name" field saying something like "Please fill in this field" (it will vary across different browsers). This is the `required` attribute — and the browser's default client-side form validation — in action.

Reviewer: We can validate it in the live sample if we want, with `allow-forms`

Author: I think this is OK as-is. What I have done is to add something at the point where we first tell the reader to make a local copy of the example, to say that you could also open it in MDN Playground if you'd prefer. It is better to have them with the example loaded in a separate tab, one way or another, rather than expecting them to scroll back up to the live example.

Reviewer: >You can't do that in the live example or the MDN Playground version.

Yeah you just get a 404 but you can't see the address / URL

> 

Author: Yeah, why not.

> 

Author: done
